docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -1,134 +1,210 @@
+     from the sidebar.
+</TabItem>
+</Tabs>
+
+### Integrate with external data sources {#integrate-external-services}
+
+For integration with external data sources, Aiven for ClickHouse offers two methods:
+
+- [Managed databases](/docs/products/clickhouse/concepts/data-integration-overview.md#managed-databases-integration)
+- [Managed credentials](/docs/products/clickhouse/concepts/data-integration-overview.md#managed-credentials-integration)
+
+To create an integration with an external data source in the **Data service integrations**
+wizard:
+
+1. Select a type of an external data service to integrate with. See
+   [Limitations](/docs/products/clickhouse/concepts/data-integration-overview#limitations)
+   for supported service types.
+1. Select an external data service of the chosen type from the list of services available
+   for integration and click **Continue**
+1. Select an integration method, either **Managed databases**  or **Managed credentials**,
+   and click **Continue**.
+1. Integrate with managed databases or with managed credentials:
+
+<Tabs groupId="group2">
+<TabItem value="1" label="Integrate with managed databases" default>
+The managed databases integration uses databases engines and, when enabled,
+automatically creates databases in your Aiven for ClickHouse, where you can ingest your
+external data.
+
+1. Select **Managed databases**, and click **Continue**.
+1. Click **Add databases** to have custom databases created along with the integration, or
+   click **Enable without databases** to integrate with no custom databases created.
+
+   :::tip
+   If you choose to have databases created automatically, you can populate them with your
+   external data using a query similar to the following:
+
+   ```sql
+   SELECT *
+   FROM ext-postgresql-resource-name.your-pg-table-name
+   ```
+
+   :::
+
+</TabItem>
+<TabItem value="2" label="Integrate with managed credentials">
+The managed credentials integration supports storing connection parameters in Aiven
+and allows you to create tables for your external data. The stored connection parameters
+are automatically seeded in your external data queries.
+
+1. Select **Managed credentials**, and click **Continue** > **Enable**.
+1. Create tables using
+   [table engines](/docs/products/clickhouse/reference/supported-table-engines), for
+   example the PostgreSQL engine:
+
+   ```sql
+   CREATE TABLE default.postgresql_table
+   (
+      `float_nullable` Nullable(Float32),
+      `str` String,
+      `int_id` Int32
+   )
+   ENGINE = PostgreSQL(postgres_creds);
+   ```
+
+   :::tip
+   For details on how to use different table engines for integrations with external
+   systems, see the
+   [upstream ClickHouse documentation](https://clickhouse.com/docs/en/engines/table-engines/integrations).
+   :::
+
+You can access your credentials storage by passing your external service name in the
+following query:
+
+```sql
+SELECT data
+FROM postgresql(ext-postgresql-resource-name,
```

Comment:
AFAIK we can use also `PostgreSQL` instead of `postgresql`. Looks more consistent. Please let me know when those function names are different @aris-aiven 

Lets use also `*` instead of `data` here